projects/onto-search/src/lib/onto-search-field/models/search-model.ts

```diff
@@ -1,3 +1,7 @@
+/**
+ * Interface defining the minimal search response parameters.
+ * Search response  must implement this interface in order response data to be processed.
+ */
```

Comment:
We did a mistake with the renaming of the component. It's kind of inconsistent now. You have a SearchFieldComponent but SearchConfiguration and SearchModel. The renaming is not applied everywhere and leads to confusion.